README.md

```diff
@@ -1,38 +1,185 @@
 
 # pillar2-acceptance-tests
 
-Acceptance tests for Pillar2 Application Features
-
+####Acceptance tests for Pillar2 Application Features
+This repository is the automated acceptance tests for Pillar2 Application which include below pages:
```

Comment:
Can we change this to something like:

These are the acceptance tests for the Pillar 2 (Global Minimum Tax) service. They test the user journey for Registering, Subscription and Maintaining Pillar 2 tax for MNEs. The service is tested on the Jenkins build pipeline for tests that run through the full journey pages included below: